api/include/mfxdefs.h

```diff
@@ -50,6 +50,29 @@ extern "C"
 {
 #endif /* __cplusplus */
 
+#if defined(_MSC_VER)
+    #define MFX_PRAGMA_IMPL(x) __pragma(x)
+#else
+    #define MFX_PRAGMA_IMPL(x) _Pragma(#x)
+#endif
+
+#define MFX_PACK_BEGIN_X(x) MFX_PRAGMA_IMPL(pack(push, x))
+
+#if defined(_WIN64) || defined(__LP64__)
```

Comment:
Do you have some links to confirm that all linux compilers define these __LP64__ and other macro defs?

Reply:
Well, I checked sources of gcc/clang and they do (unfortunately didn't find it in docs)